test/evergreen.yml

```diff
@@ -2744,6 +2748,20 @@ buildvariants:
       PATH=/opt/mongodbtoolchain/v3/bin:$PATH
   tasks:
     - name: many-dhandle-stress-test
+
+- name: many-collection-tests
+  display_name: "Many collection tests"
+  batchtime: 480 # 3 times a day
+  run_on:
+  - ubuntu1804-wt-large
+  expansions:
+    make_command: PATH=/opt/mongodbtoolchain/v3/bin:$PATH make
+    posix_configure_flags:
+      --enable-silent-rules --enable-python --enable-zlib --enable-snappy
+      --enable-strict --enable-static --prefix=$(pwd)/LOCAL_INSTALL
+    test_env_vars:
```

Comment:
@etienneptl Just realized the whole purpose of this new build variant is to run on a different distro. Actually we can reuse the existing `large-scale-tests` build variant and overload the distro for a given task using `distro:` argument setting.

Reply:
Thanks, looks way better, see c17574a